test/common/dailies.coffee

```diff
@@ -49,8 +49,8 @@ newUser = (addTasks=true)->
       user.ops.addTask {body: {type: task, id: shared.uuid()}}
   user
 
-cron = (usr) ->
-  usr.lastCron = moment().subtract(1,'days')
+cron = (usr, missedDays=1) ->
+  usr.lastCron = moment().subtract(missedDays,'days')
```

Comment:
@crookedneighbor Is this okay with you? It's to let us test cron after more than one day is missed.